app/components/candidate_interface/carried_over_content_component.rb

```diff
@@ -0,0 +1,29 @@
+module CandidateInterface
+  class CarriedOverContentComponent < ViewComponent::Base
+    def initialize(application_form:)
+      @application_form = application_form
+    end
+
+    def academic_year
+      CycleTimetable.cycle_year_range(recruitment_cycle_year - 1)
```

Comment:
I think `:cycle_year_range` is adequate because it is just formatting whatever year you are giving it.  

It is tempting to say that  `:academic year`, would mean `current_year to current_year + 1`, but in this case, we want the academic year for whatever year was *before* the current_application, which can be for *next* year if you've carried over between cycles, or it could be for this year if you carried over later. 

The context for what academic year we want is in this component, not the `CycleTimetable`, so I don't think a method there would be helpful.